UI/window-basic-adv-audio.cpp

```diff
@@ -224,9 +128,13 @@ void OBSBasicAdvAudio::SetVolumeType(bool percent)
 
 	config_set_int(GetGlobalConfig(), "BasicWindow", "AdvAudioVolumeType",
 		       (int)type);
+
+	ui->usePercent->blockSignals(true);
+	ui->usePercent->setChecked(checked);
+	ui->usePercent->blockSignals(false);
```

Comment:
```suggestion
```

This likely shouldn't be within the on_usePercent_toggled function, as the intent is that this function is called when the checkbox is toggled.

Instead, `blockSignals` should be used in the init function (surrounding a `ui->usePercent->setChecked()` call), and this bit of code